types/react-select/src/Select.d.ts

```diff
@@ -286,7 +285,11 @@ export default class Select<
     focusValue(direction: 'previous' | 'next'): void;
 
     focusOption(direction: FocusDirection): void;
-    setValue: (newValue: ValueType<OptionType, IsMulti>, action: ActionTypes, option?: OptionType) => void;
+    setValue: (
+        newValue: ValueType<OptionType, IsMulti>,
+        action: 'select-option' | 'deselect-option',
```

Comment:
can you make this union an exported type?

I think it would be best to make an enum for ActionMeta types and then use `keyof typeof ActionMetaTypes` where `ActionMetaTypes` is the relevant enum.

Then export both.